package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "validate-npm-package-name",
-  "version": "3.0.0",
```

Comment:
This change should be made during the release process and not as part of the PR

Reply:
@ghostdevhv sorry, I haven't really explained this well enough. 

The idea is that the PR is approved and merged and then after merging you update package.json (or someone), publish the package and then just commit the version number change directly to main.

Reply:
@Soph : Sure will revert this changes.